packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/lib/feature_profile/views/profile_page.dart

```diff
@@ -37,6 +40,20 @@ class ProfilePage extends StatelessWidget {
                     .push(const NotificationsRoute()),
               ),
             ),
+            SizedBox(
+              height: context.designSystem.spacing.xl0,
+            ),{{#enable_change_language}}
+            LanguagePickerButton(
+              onChanged: (language) => context
+                  .read<CoordinatorBlocType>()
+                  .events
+                  .setCurrentLanguage(language),
+              padding: context.designSystem.spacing.xl0,
+              buttonText: context.l10n.changeLanguage,
+              translate: (model) => model.key == _bulgarian
```

Comment:
Same as above, there is a need of a scalable solution

Reply:
I have used and extension method asText() to translate the string